bin/oay/Cargo.toml

```diff
@@ -46,7 +46,7 @@ axum = "0.6"
 bytes = { version = "1.4.0", optional = true }
 chrono = "0.4.28"
 clap = { version = "4", features = ["cargo", "string"] }
-dav-server = { version = "0.5.5", optional = true }
+dav-server = { version = "0.5.7", optional = true }
```

Comment:
We already use `0.5.7` in `Cargo.lock`, we don't need to bump `Cargo.toml` by hand.